packages/ocamlfind/ocamlfind.1.6.2/files/ocaml

```diff
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+BINDIR=$(dirname $(command -v ocamlc))
+$BINDIR/ocaml -I $OCAML_TOPLEVEL_PATH $*
```

Comment:
And quote everything properly. `"$@"` not `$*`, `-I "$OCAML_TOPLEVEL_PATH"` not `-I $OCAML_TOPLEVEL_PATH`,...